models/Element/Traits/VersionDaoTrait.php

```diff
@@ -24,15 +24,17 @@
 trait VersionDaoTrait
 {
     /**
-     * Get latest available version, using $force always returns a version no matter if it is the same as the published one
+     * Get latest available version, using $includingPublished to also consider the published one
      *
      * @param int|null $userId
+     * @param bool $includingPublished
      *
      * @return Version|null
      */
-    public function getLatestVersion($userId = null)
```

Comment:
when this signature was updated, should this line have been updated as well: https://github.com/pimcore/pimcore/blob/e04935c09918a762572e6e17e0e8461c7b37ed33/models/DataObject/Concrete.php#L36

Reply:
@cancan101 I created a PR: https://github.com/pimcore/pimcore/pull/17059